### ISSUE TITLE
Flashing marker sizes in Firefox

### ISSUE THREAD

**Issue by pkgw** [aperiosoftware/aas-timeseries#46]
When I use the timeseries plots in Firefox (69.0.1, running on Linux), the sizes of the markers jump around when I pan and zoom, as in this animated GIF:

![sizes2](https://user-images.githubusercontent.com/59598/65627249-f59b5200-df9c-11e9-9ea8-32b13274896a.gif)

If I zoom in and out enough, the flashing seems to stop, with the markers settling on the larger point size. @augustfly sees the same thing in Firefox on Mac.

This is a pretty unpleasant effect while it happens.

**Comment by astrofrog** [comment]
This is related to a switch between showing markers individually and a raster map, when there are enough points. I agree the switching shouldn't happen frequently like this.

@slowe has been exploring the possibility of using WebGL which might help make the switching unnecessary, so I'll discuss with him what the best way forward is.

**Comment by pkgw** [comment]
Agreed that probably switching should happen less often, but the size difference is quite large — is it somehow not possible to make the marker sizes align between the two rendering methods?

Also, FWIW, @augustfly mentioned to me that he's seeing generally severe performance problems with this example plot on Chrome/macOS. If he's not being bitten by some extremely unusual circumstance, that might be another reason to look into WebGL.

**Comment by slowe** [comment]
Actually the switch isn't it going to a raster map. That only occurs during a zoom. The rendering after zoom was given two modes because there had been a complaint that the rendering could sometimes be too slow. The simple render kicks in if it is too slow. You are getting oscillations between the two types of render as the processing time must be near the transition point. I could just remove the "quick" render to stop this.
